python/Makefile

```diff
@@ -36,7 +36,7 @@ test-integration:
 	docker-compose -f dev/docker-compose-integration.yml kill
 	docker-compose -f dev/docker-compose-integration.yml build
 	docker-compose -f dev/docker-compose-integration.yml up -d
-	sleep 20
+	sleep 30
```

Comment:
Based on https://github.com/apache/iceberg/pull/7163#issuecomment-1479219168, https://github.com/apache/iceberg/pull/7163#discussion_r1145283379, it seems sometimes the iceberg-spark docker need longer time to finish table creation. Extending the wait time to 30s to ensure the integration test can pass.